inc/checks/class-cache-flush.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace runcommand\Doctor\Checks;
+
+use WP_CLI;
+use RecursiveDirectoryIterator;
+use RecursiveIteratorIterator;
+
+/**
+ * Detects the number of occurrences of the `wp_cache_flush()` function.
+ */
+class Cache_Flush extends File_Contents {
+
+	public function run() {
+
+		// Path to wp-content directory.
+		$wp_content_dir = defined( 'WP_CONTENT_DIR' ) ? WP_CONTENT_DIR : ABSPATH . 'wp-content';
+		$directory = new RecursiveDirectoryIterator( $wp_content_dir, RecursiveDirectoryIterator::SKIP_DOTS );
+		$iterator = new RecursiveIteratorIterator( $directory, RecursiveIteratorIterator::CHILD_FIRST );
+
+		// Regex to match.
+		$this->regex = 'wp_cache_flush()';
+
+		foreach ( $iterator as $file ) {
+			$this->check_file( $file );
```

Comment:
@danielbachhuber , one concern is that this also works for string like "wp_cache_flush", which might be part of a comment.

Reply:
Can you escape your regex so only `wp_cache_flush()` is found?

```
wp_cache_flush\(\)
```